var/spack/repos/builtin/packages/atom-dft/package.py

```diff
@@ -0,0 +1,53 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+import shutil
+
+
+class AtomDft(MakefilePackage):
+    """ATOM is a program for DFT calculations in atoms and pseudopotential
+       generation."""
+
+    homepage = "https://departments.icmab.es/leem/siesta/Pseudopotentials/"
+    url      = "https://departments.icmab.es/leem/siesta/Pseudopotentials/Code/atom-4.2.6.tgz"
+
+    version('4.2.6', 'c0c80cf349f951601942ed6c7cb0256b')
+
+    depends_on('libgridxc')
+    depends_on('xmlf90')
+
+    def edit(self, spec, prefix):
+        shutil.copyfile('arch.make.sample', 'arch.make')
+        makefile = FileFilter('arch.make')
+        makefile.filter('XMLF90_ROOT=.*', 'XMLF90_ROOT = %s' %
+                        spec['xmlf90'].prefix)
+        makefile.filter('GRIDXC_ROOT=.*', 'GRIDXC_ROOT = %s' %
+                        spec['libgridxc'].prefix)
+        makefile.filter('FC=.*', 'FC = fc')
```

Comment:
Couldn't you avoid these edits by adding `XMLF90_ROOT`, `RIDXC_ROOT` and `FC` to `build_targets`?

Reply:
Thanks, I just updated the pull request. I wasn't aware of this functionality before.